packages/discord.js/typings/index.d.ts

```diff
@@ -5559,12 +5600,17 @@ export type ThreadChannelType =
   | ChannelType.GuildPublicThread
   | ChannelType.GuildPrivateThread;
 
-export interface ThreadCreateOptions<AllowedThreadType> extends StartThreadOptions {
+export interface GuildTextThreadCreateOptions<AllowedThreadType> extends StartThreadOptions {
   startMessage?: MessageResolvable;
   type?: AllowedThreadType;
   invitable?: AllowedThreadType extends ChannelType.GuildPrivateThread ? boolean : never;
 }
 
+export interface GuildForumThreadCreateOptions extends StartThreadOptions {
+  message: BaseMessageOptions | MessagePayload;
```

Comment:
From looking at [the documentation](https://discord.com/developers/docs/resources/channel#start-thread-in-forum-channel-forum-thread-message-params-object), this needs to also include stickers and flags. `BaseMessageOptions` does not have this, so I believe a new type or interface needs to be created that accounts for this?